src/components/file-uploader/FileUploader.tsx

```diff
@@ -70,12 +70,20 @@ const FileUploader: FC<FileUploaderProps> = ({
     </ListItem>
   ))
 
+  const acceptableFileTypes = validationData.filesTypes.join(', ')
+
   const uploadButton = (
     <Button component={ComponentEnum.Label} sx={sx.button} variant={variant}>
       {isImages && <CloudUploadIcon sx={styles.icon} />}
       {buttonText}
       {icon}
-      <input hidden multiple onChange={addFiles} type={InputEnum.File} />
+      <input
+        accept={acceptableFileTypes}
+        hidden
+        multiple
+        onChange={addFiles}
+        type={InputEnum.File}
+      />
```

Comment:
If I upload an image, filename of which has more than 55 characters, i receive an error. It is expected behaviour. If I try to upload same file one more time, I don't receive an error as you can see in the video below:

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/5bdde3ac-972a-46b8-aa71-35af86a0ef21

Please consider fixing it within current PR or creating new issue for it

Reply:
The issue for this bug has already been created: #1727.
That's why I think it is better to fix this bug in that issue.

Reply:
Okay, I did not remember about it. Thanks!